uktrade/data-hub-frontend
# Remove request dependency

## Description of change

This PR fixes a bug in the `axios-hawk-request` that was causing issues with the data store service. This has enabled the removal of the old `hawk-request` and the `request` dependency itself.

This branch has been deployed to UAT and manually tested there and no issues have been identified. Each of the files using the old `hawk-request` were individually changed and individually deployed to UAT without any issues.

Note - the legacy `hawk-request` files were deleted and the `axios-hawk-request` files were renamed. However, GitHub has shown this change as deleting the axios files and updating the legacy hawk ones, which is incorrect (this is clearer if you look at the individual commits rather than the full 'Files changed' window).

## Test instructions
Add a company and select a company from the list. The company region should display correctly and without errors.

Everything else should work as before.

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
